docs/commands.md

```diff
@@ -248,13 +248,13 @@ If this command is used to add data to an existing timeseries, `retentionTime` a
 Create a compaction rule.
 
 ```sql
-TS.CREATERULE sourceKey destKey AGGREGATION aggregationType timeBucket
+TS.CREATERULE sourceKey destKey AGGREGATION aggregationType bucketDuration
```

Comment:
@LiorKogan  We need to change the code to support this change right? (CC: @danni-m )

Reply:
No, I just changed the name of the parameter, to make it more clear....